gen/ad_admin_user_mu_ucn

```diff
@@ -115,4 +115,13 @@ for my $login (@logins) {
 
 close(FILE);
 
+sub checkBase64 {
```

Comment:
This added method is not actually used by the script, so I'm not sure we should add it.